templates/allauth/account/email.html

```diff
@@ -0,0 +1,75 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Account" %}{% endblock %}
+
+{% block inner_content %}
+    <hr>
+    <h2 class="logo-font mb-4">{% trans "E-mail Addresses" %}</h2>
+    <hr>
+{% if user.emailaddress_set.all %}
```

Comment:
This `.all` reads a bit strangely in this context. I think I would go with `.exists()`.

Also, I would generally prefer to do all ORM querying in the view rather than the template.  I've had situations in the past of situations when a query in a template was accidentally put into a loop (and slowing the page load significantly) as part of what I had thought was an innocent HTML refactoring

Reply:
This is a copy of an allauth template also.